src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -126,3 +131,10 @@ def load_command_table(self, _):
         g.command('delete', 'delete', confirmation=True)
         g.command('show', 'get')
         g.command('list', 'list_by_server')
+
+    with self.command_group('postgres server vnet-rule', postgres_vnet_sdk) as g:
+         g.command('create', 'create_or_update')
+         g.command('delete', 'delete', confirmation=True)
+         g.command('show', 'get')
+         g.command('list', 'list_by_server')
+         g.generic_update_command('update')
```

Comment:
If the setter is no longer named "create_or_update" then this will not work.  This syntax for generic update only works if the getter is called "get", the setter is "create_or_update" and both of these methods come from the SDK.  If the SDK now has a patch-like update method, then we'll need to discuss the best way to move forward.

Reply:
modified to create_or_update